WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/RetryFailedMediaUploadUseCase.kt

```diff
@@ -33,7 +39,8 @@ class RetryFailedMediaUploadUseCase @Inject constructor(
                         editorMediaListener,
                         mediaModels
                     )
-                    tracker.track(Stat.EDITOR_UPLOAD_MEDIA_RETRIED)
+                    val site: SiteModel? = siteStore.getSiteByLocalId(editorMediaListener.getImmutablePost().localSiteId)
```

Comment:
It looks like this line just needs to be reformatted, the linter's failing with the following:

```
[2024-01-08T23:57:45Z] /var/lib/buildkite-agent/builds/ci-android-i-0a5985984fd2646ad-1/automattic/wordpress-android/WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/RetryFailedMediaUploadUseCase.kt:42:1: Line detected, which is longer than the defined maximum line length in the code style. [MaxLineLength]
```